### ISSUE TITLE
how to finetune the model-20170512-110547.ckpt-250000?

### ISSUE THREAD

**Issue by billtiger** [davidsandberg/facenet#533]
when i strive to finetune the model-20170512-110547.ckpt-250000 by using my own dataset(23 classes),there have a error:
InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [128,23] rhs shape= [128,44052]
I think i should modify the final layer?but i don't how to modify the final layer,can anyone help me?
thanks!

**Comment by billtiger** [comment]
@davidsandberg

**Comment by NanhaiK** [comment]
did you solve the problem? i got the same error

**Comment by hustzeyu** [comment]
I have tried 100*100, with -- image_size 100, but it shows:
InvalidArgumentError (see above for traceback): computed output size would be negative

**Comment by qf6877** [comment]
when load the weights, exclude the scope "logits" weights
